src/client/containers/collection/index.jsx

```diff
@@ -1,18 +1,29 @@
 import React, { Component } from 'react';
 import './index.css';
+import { shuffle } from 'lodash';
```

Comment:
@pkallas Actually I was referring to using this - https://www.npmjs.com/package/lodash.shuffle